calculator.go

```diff
@@ -1,13 +1,91 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
+import (
+		"fmt"
+		"math"
+)
+
 // Add takes two numbers and returns the result of adding them together.
 func Add(a, b float64) float64 {
-return a + b
+	return a + b
 }
 
 // Subtract takes two numbers and returns the result of subtracting the second
 // from the first.
 func Subtract(a, b float64) float64 {
-	return b - a
+	return a - b
+
 }
+
+// Attempting to break the test
```

Comment:
Don't forget to remove these temporary comments once the program is more or less finished, and replace them with documentation comments according to Go's conventional format:

https://golang.org/doc/effective_go.html#commentary